products/appengine/api.yaml

```diff
@@ -63,3 +63,537 @@ objects:
           - UNSPECIFIED_ACTION
           - ALLOW
           - DENY
+  - !ruby/object:Api::Resource
+    name: 'StandardAppVersion'
+    description: |
+      Standard App Version resource to create a new version of standard GAE Application.
+      Currently supporting Zip and File Containers.
+      Currently does not support async operation checking.
+    base_url: 'apps/{{project}}/versions'
+    self_link: 'apps/{{project}}/versions/{{version_id}}'
+    update_verb: :PATCH
+    update_mask: true
+    references: !ruby/object:Api::Resource::ReferenceLinks
+      guides:
+        'Official Documentation':
+          'https://cloud.google.com/appengine/docs/admin-api/deploying-overview'
+      api: 'https://cloud.google.com/appengine/docs/admin-api/reference/rest/v1/apps.services.versions'
+    parameters:
+# Not sure if this parameter is required. Should be removed      
+      - !ruby/object:Api::Type::String
+        name: 'name'
+        description: |
+          Full path to the Version resource in the API. Example, apps/myapp/services/default/versions/v1.
+    properties:
+# The id parameter is required and overridden as 'version_id'   
+      - !ruby/object:Api::Type::String
+        name: 'id'
+        description: |
+          Relative name of the version within the service. Example v1. Version names can contain only lowercase letters, numbers, or hyphens. Reserved names,"default", "latest", and any name with the prefix "ah-".
+# inboundServices      
+      - !ruby/object:Api::Type::Enum
+        name: 'inboundServices'
+        description: |
+          Before an application can receive email or XMPP messages, the application must be configured to enable the service.
+        values:
+          - :INBOUND_SERVICE_UNSPECIFIED
+          - :INBOUND_SERVICE_MAIL
+          - :INBOUND_SERVICE_MAIL_BOUNCE	
+          - :INBOUND_SERVICE_XMPP_ERROR
+          - :INBOUND_SERVICE_XMPP_MESSAGE	
+          - :INBOUND_SERVICE_XMPP_SUBSCRIBE
+          - :INBOUND_SERVICE_XMPP_PRESENCE
+          - :INBOUND_SERVICE_CHANNEL_PRESENCE
+          - :INBOUND_SERVICE_WARMUP     
+# instanceClass     
+      - !ruby/object:Api::Type::String
+        name: 'instanceClass'
+        description: |
+          Instance class that is used to run this version. Valid values are
+          AutomaticScaling F1, F2, F4, F4_1G
+          ManualScaling or BasicScaling B1, B2, B4, B8, B4_1G
+          Defaults to F1 for AutomaticScaling and B1 for ManualScaling or BasicScaling.
+        required: false        
+# network paramter is missing.
+# zones      
+      - !ruby/object:Api::Type::Array
+        name: 'zones'
+        item_type: Api::Type::String
+        description: 'The Google Compute Engine zones that are supported by this version in the App Engine flexible environment.'
```

Comment:
For descriptions, I'd suggest always using the `description: |` form. We treat them as preformatted markdown, while this treats them as a string w/ the newline removed.